packages/core/src/components/HelpDrawer/__snapshots__/HelpDrawer.test.jsx.snap

```diff
@@ -27,28 +27,36 @@ exports[`HelpDrawer renders a snapshot 1`] = `
     </div>
   </div>
   <div
-    className="ds-c-help-drawer__body ds-u-md-font-size--small ds-u-lg-font-size--base ds-u-padding--2"
+    className="ds-c-help-drawer__body"
   >
-    <p>
-      content
-    </p>
-  </div>
-  <div
-    className="ds-c-help-drawer__footer ds-u-fill--primary-alt-lightest ds-u-md-font-size--small ds-u-lg-font-size--base ds-u-padding--2"
-  >
-    <h4
-      className="ds-text ds-u-margin--0"
-    >
-      Footer title
-    </h4>
     <div
-      className="ds-text ds-u-margin--0"
+      className="ds-c-help-drawer__content-container"
     >
-      <div>
+      <div
+        className="ds-c-help-drawer__content ds-u-md-font-size--small ds-u-lg-font-size--base ds-u-padding--2"
+      >
         <p>
-          Some footer content
+          content
         </p>
       </div>
+      <div
```

Comment:
I think the footer div needs to be moved to after the closing `ds-c-help-drawer__body` div so it lives outside of the body. Right now the footer is floating up the page. This is in Chrome and IE 11. 
![Screen Shot 2020-01-14 at 12 24 02 PM](https://user-images.githubusercontent.com/1449852/72366834-0ea14d80-36c9-11ea-9ba8-fc165d2e6a91.png)

Reply:
Hm yeah, the tricky thing is that if the help drawer body is very long, we want the footer to be hidden until you scroll all the way down. [This is the existing behavior at least](https://gyazo.com/473039de8365b93b577b8b23a33a43f9), I'm also open to changing it